src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -5,12 +5,18 @@ public class AuthenticationService {
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private final UserService service = new UserService();
+
     public boolean login(String email, String password) {
+        if (service.findByEmail(email) != null) {
+            return service.findByEmail(email).getPassword().equals(password);
+        }
         return false;
```

Comment:
you call findByEmal twice. save the result of the single call to a local variable, and reuse it. Try to use only one return in this method